### PR TITLE
Contributing

### PR DESCRIPTION
**Pull request by pevik**
Add section in README.md and issue and pull request templates.

### CODE REVIEW

**Comment by pevik** [comment]
@gobenji FYI

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+If fixing bug, please document how to reproduce it.
```
Comment (okias) [inline .github/issue_template.md:1]: If fixing a bug, please document how to reproduce it.

Please always test the latest master.

Finding the commit which introduced the problem helps (bisecting).

Document the kernel and distribution that were used.




If adding a new feature, explain why it's useful.





Tests should ideally use network namespaces to not interfere with the rest of the system.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+If fixing bug, please document how to reproduce it.
```
Comment (okias) [inline .github/pull_request_template.md:1]: If fixing a bug, please document how to reproduce it. Tests should ideally use network namespaces to not interfere with the rest of the system.

If adding a feature, please describe why it's useful to add it.

Commits should be signed: Your Name [me@example.org](mailto:me@example.org), see https://www.kernel.org/doc/html/latest/process/submitting-patches.html#sign-your-work-the-developer-s-certificate-of-origin


Although the coding style for most tools is ancient, new code should follow the Linux kernel coding style. See https://www.kernel.org/doc/html/latest/process/coding-style.html.

File: README.md [hunk 3]
```diff
@@ -22,6 +21,17 @@ Build dependencies are listed in scripts in
 - [uClibc-ng](https://uclibc-ng.org/)
 - [musl](https://musl.libc.org/)
 
+## Contributing
```
Comment (okias) [inline README.md:24]: same here

**Review by okias — COMMENTED** [review]
I like the change, but for the English language, use some autocorrecting service, if possible (I use.... Place Ad here... https://quillbot.com/grammar-check )

**Comment by pevik** [comment]
@okias I added tiny changes, can you have a final quick look? Can I merge with your RBT?

**Comment by okias** [comment]
> @okias I added tiny changes, can you have a final quick look? Can I merge with your RBT?

yeah! Thank you!

**Pull request closed without merging**